downstairs/src/region.rs

```diff
@@ -1316,12 +1373,14 @@ impl Region {
             "Repair files downloaded, move directory {:?} to {:?}",
             copy_dir, rd
         );
-
-        // XXX fsync the parent directory (the extent dir)
```

Comment:
I fsync after the rename, so I did not think I needed an fsync before.  The `save_stream_to_file` also does an fsync on the file itself, so that piece should be covered.